test/types/pino.test-d.ts

```diff
@@ -0,0 +1,267 @@
+import P, { pino } from "../../";
+import { IncomingMessage, ServerResponse } from "http";
+import { Socket } from "net";
+
+const log = pino();
+const info = log.info;
+const error = log.error;
+
+info("hello world");
+error("this is at error level");
+info("the answer is %d", 42);
+info({ obj: 42 }, "hello world");
+info({ obj: 42, b: 2 }, "hello world");
+info({ obj: { aa: "bbb" } }, "another");
+setImmediate(info, "after setImmediate");
+error(new Error("an error"));
+
+const writeSym = pino.symbols.writeSym;
+
+const testUniqSymbol = {
+    [pino.symbols.needsMetadataGsym]: true,
+}[pino.symbols.needsMetadataGsym];
+
+const log2: P.Logger = pino({
+    name: "myapp",
+    safe: true,
+    serializers: {
+        req: pino.stdSerializers.req,
+        res: pino.stdSerializers.res,
+        err: pino.stdSerializers.err,
+    },
+});
```

Comment:
Does this use tabs for indentation? Maybe can we have this conform to the rest?

Reply:
It already uses spaces.